src/libs/actions/Policy.js

```diff
@@ -375,7 +381,7 @@ function addMembersToWorkspace(memberLogins, welcomeNote, policyID, betas) {
         // Escape HTML special chars to enable them to appear in the invite email
         welcomeNote: _.escape(welcomeNote),
         policyID,
-        optimisticReportIDs: JSON.stringify(membersChats.optimisticReportIDs),
+        reportCreationData: JSON.stringify(membersChats.reportCreationData),
```

Comment:
crap just realizing I might have screwed up the WAF rule for this 🤦 going to see about testing real quick.

Reply:
and yes confirmed I did this wrong, let me fix it.